app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by name. This could lead to issues if multiple people have the same name, as the dictionary will overwrite the previous entry. Consider using a unique identifier for each person, such as an ID, to avoid this problem.